sdk/cosmos/azure_data_cosmos/tests/framework/test_account.rs

```diff
@@ -91,14 +106,8 @@ impl TestAccount {
             return Err("invalid connection string, missing 'AccountKey'".into());
         };
 
-        let context_id = format!(
-            "{}_{}_{}",
-            context.name(),
-            OffsetDateTime::now_utc().format(format_description!(
-                "[year]_[month]_[day]T[hour]_[minute]_[second]"
-            ))?,
-            Uuid::new_v4().as_simple()
-        );
+        // We need the context_id to be constant, so that record/replay work.
+        let context_id = context.name().to_string();
 
         TRACING.call_once(|| {
```

Comment:
You may not need this since the test proxy code does it using `RUST_LOG` as well. You might try deleting this code and running your tests e.g.,

```bash
RUST_LOG=debug,azure_data_cosmos=trace cargo test -p azure_data_cosmos
```

Or something like that. You typically don't want to trace everything because `reqwest` and `hyper` are noisy.